model/webhook.go

```diff
@@ -6,9 +6,43 @@ package model
 
 import (
 	"encoding/json"
+	"fmt"
 	"io"
+	"os"
+	"strings"
 )
 
+const (
+	WebhookHeaderEnvironmentValuePrefix = "env:"
+)
+
+// ParseHeadersFromStringMap parses the headers from a string map, finding and returning correct
+// header values for the items that require getting values from environment variables
+// This is also used in event subscription headers
+func ParseHeadersFromStringMap(sm *StringMap) (map[string]string, error) {
+	if sm == nil {
+		return map[string]string{}, nil
+	}
+	headers := make(map[string]string, len(*sm))
+	var err error
+	for key, value := range *sm {
+		headerValue := value
+		if strings.HasPrefix(value, WebhookHeaderEnvironmentValuePrefix) {
+			headerValue = os.Getenv(strings.TrimPrefix(value, WebhookHeaderEnvironmentValuePrefix))
```

Comment:
I ended up going this way because it wasn't so difficult to understand (IMO) and it made easier using the CLI (single flag) and manually editing the database (less payload). I'm not against the other way, but since this is more internal than external I'd rather use `--headers A=B,C=D` than `--header A=B --header-from-env C=D` or something like that. I could also use the same input and just internally marshall into that other format, but it would only add complexity and I don't see much benefit at the moment. 

If we already have some plans on introduce other ways of setting header values, having a new structure is more extensible and I would go for that (before we start using this one).

Any thoughts?

Reply:
No, I really can't foresee other use-cases so I think this will work just fine. I just wanted to check in to see what trade-offs you ran into. Let's go with the simpler implementation!